packages/evergreen-checkbox/src/components/Checkbox.js

```diff
@@ -0,0 +1,95 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import Box from 'ui-box'
+import CheckboxAppearances from '../styles/checkbox-appearances'
+import colors from 'evergreen-colors'
+import { Text } from 'evergreen-typography'
+
+const CheckIcon = ({ fill = 'currentColor', ...props }) => (
+  <svg width={10} height={7} viewBox="0 0 10 7" {...props}>
+    <path
+      fill={fill}
+      fillRule="evenodd"
+      d="M4 4.586L1.707 2.293A1 1 0 1 0 .293 3.707l3 3a.997.997 0 0 0 1.414 0l5-5A1 1 0 1 0 8.293.293L4 4.586z"
+    />
+  </svg>
+)
+
+export default class Checkbox extends PureComponent {
```

Comment:
I would make primitives like this a Component rather than a PureComponent because PureComponent has performance overhead. Higher level components like those in the app should be making the PureComponent optimisation.